src/pudl/analysis/record_linkage/name_cleaner.py

```diff
@@ -266,4 +266,11 @@ def apply_name_cleaning(
         Returns:
             df (dataframe): the clean version of the input dataframe
         """
+        if isinstance(df, pd.DataFrame) and len(df.columns) > 1:
+            clean_df = pd.DataFrame()
+            for col in df.columns:
+                clean_df = pd.concat(
+                    [clean_df, df[col].apply(self.get_clean_data)], axis=1
+                )
+            return clean_df
```

Comment:
This allows the name cleaner to work on a dataframe with multiple columns, so if you pass in a dataframe with `plant_name_ferc1` and `plant_name_eia` then it will clean both.